### DISCUSSION TITLE
AWS4 Signer producing incorrect signature

### DISCUSSION THREAD

**Discussion by dmollin** [aws/aws-sdk-java-v2#4963] [Q&A] [answered]
Hello, I am running into the same issue as described in [Issue#4550](https://github.com/aws/aws-sdk-java-v2/issues/4550), with the same workflow/architecture. Looks like the bug was closed but can someone share what the fix to the signer or code logic was?

**Comment by debora-ito** [comment, marked as answer]
@dmollin There was no change for that issue. Are you also using GCP or other third-party solution? If you are, try reaching out to their support team first. 

If you are using a AWS endpoint directly, please open new Github issue and provide all the details we ask in the template.
